2025-02-01 18:43:58,043 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 18:43:58,046 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:43:58] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 18:44:08,684 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:44:08] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 18:44:08,716 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:44:08] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 18:44:08,720 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:44:08] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 18:44:08,727 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:44:08] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:44:08,743 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:44:08] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:44:08,774 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:44:08] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:44:08,865 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:44:08] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:46:08,382 - INFO - [app.py:30] - Current User: pc
2025-02-01 18:46:08,382 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 18:46:08,382 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 18:46:08,382 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 18:46:08,383 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 18:46:08,383 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 18:46:08,383 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 18:46:08,389 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 18:46:08,389 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 18:46:08,510 - INFO - [app.py:30] - Current User: pc
2025-02-01 18:46:08,510 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 18:46:08,510 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 18:46:08,510 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 18:46:08,511 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 18:46:08,511 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 18:46:08,511 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 18:46:08,515 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 18:46:08,515 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 18:46:12,159 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:12] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 18:46:12,179 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:12] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 18:46:12,186 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:12] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:46:12,187 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:12] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 18:46:12,198 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:12] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:46:12,230 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:12] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 18:46:12,301 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:12] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 18:46:15,828 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 18:46:15,828 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 18:46:15,829 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:46:15] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 18:47:10,160 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 18:47:10,182 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:47:10] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 18:54:59,854 - INFO - [app.py:30] - Current User: pc
2025-02-01 18:54:59,854 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 18:54:59,854 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 18:54:59,855 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 18:54:59,855 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 18:54:59,856 - INFO - [app.py:68] - Directory Details:
2025-02-01 18:54:59,856 - INFO - [app.py:69] -   Path: /var/www/html/vCard/vcards
2025-02-01 18:54:59,856 - INFO - [app.py:70] -   Current Permissions: 0o777
2025-02-01 18:54:59,856 - INFO - [app.py:71] -   Owner UID: 0
2025-02-01 18:54:59,856 - INFO - [app.py:72] -   Group GID: 0
2025-02-01 18:54:59,857 - ERROR - [app.py:83] - Could not change vcards directory permissions: Command '['chmod', '777', '/var/www/html/vCard/vcards']' returned non-zero exit status 1.
2025-02-01 18:54:59,863 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 18:54:59,864 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 18:54:59,983 - INFO - [app.py:30] - Current User: pc
2025-02-01 18:54:59,983 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 18:54:59,983 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 18:54:59,983 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 18:54:59,984 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 18:54:59,984 - INFO - [app.py:68] - Directory Details:
2025-02-01 18:54:59,984 - INFO - [app.py:69] -   Path: /var/www/html/vCard/vcards
2025-02-01 18:54:59,984 - INFO - [app.py:70] -   Current Permissions: 0o777
2025-02-01 18:54:59,984 - INFO - [app.py:71] -   Owner UID: 0
2025-02-01 18:54:59,984 - INFO - [app.py:72] -   Group GID: 0
2025-02-01 18:54:59,986 - ERROR - [app.py:83] - Could not change vcards directory permissions: Command '['chmod', '777', '/var/www/html/vCard/vcards']' returned non-zero exit status 1.
2025-02-01 18:54:59,990 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 18:54:59,991 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 18:55:03,139 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:55:03] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 18:55:03,158 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:55:03] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 18:55:03,161 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:55:03] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 18:55:03,163 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:55:03] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:55:03,174 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:55:03] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:55:03,201 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:55:03] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:55:03,272 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:55:03] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:12,255 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:12] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 18:56:12,274 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:12] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 18:56:12,281 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:12] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 18:56:12,283 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:12] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:12,294 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:12] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:12,322 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:12] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:12,396 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:12] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:14,156 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:14] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 18:56:14,181 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:14] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 18:56:14,186 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:14] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 18:56:14,188 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:14] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:14,197 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:14] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:14,227 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:14] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:14,232 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:14] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 18:56:18,008 - INFO - [app.py:136] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 18:56:18,008 - INFO - [app.py:144] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 18:56:18,009 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:18] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 18:56:25,987 - INFO - [app.py:172] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 18:56:26,007 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 18:56:26] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 18:56:42,993 - INFO - [_internal.py:224] -  * Detected change in '/var/www/html/vCard/app.py', reloading
2025-02-01 18:56:43,035 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 18:56:43,195 - INFO - [app.py:30] - Current User: pc
2025-02-01 18:56:43,195 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 18:56:43,195 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 18:56:43,195 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 18:56:43,196 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 18:56:43,196 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 18:56:43,196 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 18:56:43,201 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 18:56:43,202 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 18:57:16,526 - INFO - [app.py:30] - Current User: pc
2025-02-01 18:57:16,526 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 18:57:16,526 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 18:57:16,526 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 18:57:16,527 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 18:57:16,527 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 18:57:16,528 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 18:57:16,533 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 18:57:16,533 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 18:57:16,653 - INFO - [app.py:30] - Current User: pc
2025-02-01 18:57:16,653 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 18:57:16,653 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 18:57:16,653 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 18:57:16,654 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 18:57:16,654 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 18:57:16,654 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 18:57:16,658 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 18:57:16,659 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:06:33,108 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:06:33,108 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:06:33,108 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:06:33,108 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:06:33,109 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:06:33,109 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:06:33,109 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:06:33,114 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:06:33,115 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:06:33,234 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:06:33,235 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:06:33,235 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:06:33,235 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:06:33,236 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:06:33,236 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:06:33,236 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:06:33,240 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:06:33,240 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:06:41,311 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:41] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 19:06:41,337 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:41] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:06:41,344 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:41] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 19:06:41,348 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:41] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:06:41,378 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:41] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:06:41,385 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:41] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:06:41,572 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:41] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:06:44,484 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:06:44,485 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:06:44,485 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:06:44] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:07:29,506 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:07:29,527 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:07:29,527 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:07:29] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:08:09,256 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:08:09,256 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:08:09,257 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:08:09] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:11:05,474 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:11:05,474 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:11:05,474 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:11:05,474 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:11:05,475 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:11:05,475 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:11:05,475 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:11:05,480 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:11:05,481 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:11:05,597 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:11:05,597 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:11:05,597 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:11:05,597 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:11:05,598 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:11:05,598 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:11:05,598 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:11:05,602 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:11:05,603 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:11:08,051 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:08] "GET / HTTP/1.1" 200 -
2025-02-01 19:11:08,078 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:08] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 19:11:08,082 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:08] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 19:11:08,083 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:08] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:11:08,092 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:08] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:11:08,131 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:08] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:11:08,182 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:08] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:11:13,534 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:11:13,540 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:11:13,542 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:13] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:11:24,112 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:11:24,129 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:11:24,130 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:11:24] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:13:49,888 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:13:49,888 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:13:49,888 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:13:49,888 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:13:49,889 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:13:49,889 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:13:49,889 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:13:49,894 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:13:49,895 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:13:50,011 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:13:50,011 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:13:50,011 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:13:50,011 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:13:50,012 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:13:50,012 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:13:50,012 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:13:50,016 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:13:50,017 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:14:07,703 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:07] "GET / HTTP/1.1" 200 -
2025-02-01 19:14:07,736 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:07] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 19:14:07,741 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:07] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 19:14:07,743 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:07] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:14:07,752 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:07] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:14:07,779 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:07] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:14:07,785 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:07] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:14:11,311 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:14:11,311 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:14:11,312 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:11] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:14:18,134 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:14:18,153 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:14:18,154 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:14:18] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:16:18,579 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:16:18,579 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:16:18,579 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:16:18,579 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:16:18,580 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:16:18,580 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:16:18,580 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:16:18,585 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:16:18,586 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:16:18,707 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:16:18,707 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:16:18,707 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:16:18,707 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:16:18,708 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:16:18,708 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:16:18,708 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:16:18,712 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:16:18,712 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:16:23,395 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:23] "GET / HTTP/1.1" 200 -
2025-02-01 19:16:23,407 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:23] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:16:23,408 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:23] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 19:16:23,408 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:23] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 19:16:23,411 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:23] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:16:23,443 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:23] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:16:23,489 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:23] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:16:27,613 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:27] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:16:30,797 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:16:30,797 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:16:30,798 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:30] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:16:47,965 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:16:47,983 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:16:47,984 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:16:47] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:30:05,044 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:30:05,044 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:30:05,044 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:30:05,044 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:30:05,045 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:30:05,045 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:30:05,045 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:30:05,050 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:30:05,051 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:30:05,176 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:30:05,176 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:30:05,176 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:30:05,176 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:30:05,177 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:30:05,177 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:30:05,177 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:30:05,181 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:30:05,181 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:30:08,782 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:08] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 19:30:08,808 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:08] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:30:08,814 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:08] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 19:30:08,816 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:08] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:30:08,856 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:08] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:30:08,919 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:08] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:30:08,920 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:08] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:30:12,141 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:30:12,141 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:30:12,141 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:12] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:30:26,336 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:30:26,355 - WARNING - [app.py:248] - Error processing photo: 'str' object has no attribute 'name'
2025-02-01 19:30:26,355 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:30:26] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:30:39,443 - INFO - [_internal.py:224] -  * Detected change in '/var/www/html/vCard/app.py', reloading
2025-02-01 19:30:39,477 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:30:39,639 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:30:39,639 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:30:39,639 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:30:39,639 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:30:39,640 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:30:39,641 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:30:39,641 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:30:39,645 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:30:39,646 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:31:10,299 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:10] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 19:31:10,330 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:10] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 19:31:10,331 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:10] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:31:10,334 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:10] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:31:10,345 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:10] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:31:10,381 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:10] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:31:10,390 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:10] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:31:13,571 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:31:13,571 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:31:13,572 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:13] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:31:21,943 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:31:21,943 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:31:21,943 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:31:21] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:38:10,923 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:38:10,923 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:38:10,923 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:38:10,923 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:38:10,924 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:38:10,924 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:38:10,924 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:38:10,930 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:38:10,930 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:38:11,055 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:38:11,055 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:38:11,055 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:38:11,055 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:38:11,056 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:38:11,056 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:38:11,056 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:38:11,061 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:38:11,061 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:38:19,533 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:19] "GET / HTTP/1.1" 200 -
2025-02-01 19:38:19,548 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:19] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 19:38:19,549 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:19] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 19:38:19,550 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:19] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:38:19,553 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:19] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:38:19,584 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:19] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:38:19,629 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:19] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:38:23,567 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:23] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:38:27,319 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:38:27,319 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:38:27,320 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:27] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:38:52,511 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:38:52,530 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:38:52,531 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:38:52] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:42:25,343 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:42:25,343 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:42:25,343 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:42:25,343 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:42:25,344 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:42:25,344 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:42:25,344 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:42:25,349 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:42:25,349 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:42:25,465 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:42:25,465 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:42:25,465 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:42:25,465 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:42:25,466 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:42:25,466 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:42:25,466 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:42:25,470 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:42:25,471 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:42:30,811 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:30] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 19:42:30,851 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:30] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:42:30,858 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:30] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 19:42:30,860 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:30] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:42:30,880 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:30] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:42:30,899 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:30] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:42:31,044 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:31] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:42:34,405 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:42:34,405 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:42:34,406 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:34] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:42:53,559 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:42:53,581 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:42:53,582 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:42:53] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:45:36,050 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:45:36,050 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:45:36,050 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:45:36,050 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:45:36,051 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:45:36,051 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:45:36,051 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:45:36,056 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:45:36,057 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:45:36,175 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:45:36,176 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:45:36,176 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:45:36,176 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:45:36,176 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:45:36,177 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:45:36,177 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:45:36,181 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:45:36,181 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:45:39,851 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:45:39] "GET / HTTP/1.1" 200 -
2025-02-01 19:45:39,872 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:45:39] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 19:45:39,874 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:45:39] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 19:45:39,877 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:45:39] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:45:39,885 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:45:39] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:45:39,890 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:45:39] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:45:39,933 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:45:39] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:47:32,621 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:47:32,621 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:47:32,621 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:47:32,621 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:47:32,622 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:47:32,622 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:47:32,622 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:47:32,628 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:47:32,628 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:47:32,750 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:47:32,750 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:47:32,750 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:47:32,750 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:47:32,751 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:47:32,751 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:47:32,751 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:47:32,755 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:47:32,756 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:47:34,902 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:47:34] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 19:47:34,925 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:47:34] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:47:34,932 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:47:34] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:47:34,933 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:47:34] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 19:47:34,951 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:47:34] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:47:34,974 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:47:34] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 19:47:35,052 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:47:35] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 19:50:29,556 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:50:29] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 19:50:29,587 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:50:29] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:50:29,592 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:50:29] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 19:50:29,593 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:50:29] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:50:29,619 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:50:29] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:50:29,634 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:50:29] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 19:50:29,839 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:50:29] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 19:55:14,588 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:55:14,588 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:55:14,588 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:55:14,589 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:55:14,589 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:55:14,590 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:55:14,590 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:55:14,595 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:55:14,595 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:55:14,711 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:55:14,711 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:55:14,711 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:55:14,711 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:55:14,712 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:55:14,712 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:55:14,712 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:55:14,716 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:55:14,717 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:55:27,195 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:27] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 19:55:27,227 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:27] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 19:55:27,229 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:27] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 19:55:27,232 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:27] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:55:27,270 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:27] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:55:27,373 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:27] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 19:55:27,444 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:27] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:55:30,415 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:55:30,415 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:55:30,415 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:30] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:55:38,602 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:55:38,602 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:55:38,602 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:38] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:55:41,445 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:55:41,445 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:55:41,445 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:41] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:55:44,167 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 19:55:44,187 - WARNING - [app.py:228] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 19:55:44,187 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:55:44] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 19:57:00,239 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:00,240 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:00,241 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:57:00] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:57:05,309 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:05,309 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:05,309 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:57:05] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:57:09,172 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:09,172 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:09,172 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:57:09] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:57:16,271 - INFO - [app.py:111] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:16,271 - INFO - [app.py:119] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 19:57:16,271 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:57:16] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 19:59:38,399 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:59:38,399 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:59:38,399 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:59:38,399 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:59:38,400 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:59:38,400 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:59:38,400 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:59:38,405 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 19:59:38,405 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 19:59:38,530 - INFO - [app.py:30] - Current User: pc
2025-02-01 19:59:38,530 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 19:59:38,530 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 19:59:38,530 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 19:59:38,531 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 19:59:38,531 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 19:59:38,531 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 19:59:38,535 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 19:59:38,535 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 19:59:44,013 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:44] "GET / HTTP/1.1" 200 -
2025-02-01 19:59:44,040 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:44] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 19:59:44,047 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:44] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 19:59:44,048 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:44] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:59:44,089 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:44] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 19:59:44,097 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:44] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:59:44,148 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:44] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 19:59:47,719 - ERROR - [app.py:132] - Unexpected error generating vCard: 400 Bad Request: Did not attempt to load JSON data because the request Content-Type was not 'application/json'.
2025-02-01 19:59:47,720 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:47] "[35m[1mPOST /generate_vcard HTTP/1.1[0m" 500 -
2025-02-01 19:59:47,720 - ERROR - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:47] code 400, message Bad request syntax ('------WebKitFormBoundarycBXavugMkD2M2oqP')
2025-02-01 19:59:47,720 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 19:59:47] "[35m[1mNone /generate_vcard HTTP/0.9[0m" HTTPStatus.BAD_REQUEST -
2025-02-01 20:00:04,293 - ERROR - [app.py:132] - Unexpected error generating vCard: 400 Bad Request: Did not attempt to load JSON data because the request Content-Type was not 'application/json'.
2025-02-01 20:00:04,294 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:00:04] "[35m[1mPOST /generate_vcard HTTP/1.1[0m" 500 -
2025-02-01 20:00:04,294 - ERROR - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:00:04] code 400, message Bad request syntax ('------WebKitFormBoundarylqXB5H9cxS5v8J1Y')
2025-02-01 20:00:04,294 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:00:04] "[35m[1mNone /generate_vcard HTTP/0.9[0m" HTTPStatus.BAD_REQUEST -
2025-02-01 20:01:01,369 - INFO - [app.py:147] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:01:01,388 - ERROR - [app.py:248] - Error parsing vCard QnJhaGltIEFpdG91.vcf: [Errno 2] No such file or directory: '/var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf'
2025-02-01 20:01:01,389 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:01:01] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:03:30,371 - INFO - [_internal.py:224] -  * Detected change in '/var/www/html/vCard/app.py', reloading
2025-02-01 20:03:30,397 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:03:30,547 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:03:30,547 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:03:30,547 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:03:30,547 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:03:30,548 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:03:30,549 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:03:30,549 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:03:30,554 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:03:30,554 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:03:44,467 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:03:44,467 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:03:44,467 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:03:44,467 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:03:44,468 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:03:44,468 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:03:44,468 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:03:44,473 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 20:03:44,474 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:03:44,590 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:03:44,591 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:03:44,591 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:03:44,591 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:03:44,592 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:03:44,592 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:03:44,592 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:03:44,596 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:03:44,596 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:03:47,492 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:47] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 20:03:47,518 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:47] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 20:03:47,525 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:47] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 20:03:47,527 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:47] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:03:47,545 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:47] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:03:47,573 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:47] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:03:47,662 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:47] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:03:50,371 - INFO - [app.py:135] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:03:50,371 - INFO - [app.py:143] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:03:50,372 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:03:50] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:04:01,364 - INFO - [app.py:135] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:04:01,365 - INFO - [app.py:143] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:04:01,365 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:01] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:04:03,360 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:04:03,376 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:04:03,377 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:03] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:04:03,387 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:04:03,388 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:03] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:04:22,772 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:04:22,776 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:04:22,778 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:22] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:04:22,802 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:04:22,802 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:22] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:04:24,458 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:04:24,461 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:04:24,462 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:24] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:04:24,486 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:04:24,487 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:24] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:04:24,850 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:04:24,854 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:04:24,856 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:24] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:04:24,881 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:04:24,882 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:24] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:04:25,059 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:04:25,060 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:04:25,060 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:25] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:04:25,074 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:04:25,075 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:04:25] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:05:31,768 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:05:31,770 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:05:31,770 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:05:31] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:05:31,803 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:05:31,804 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:05:31] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:06:10,518 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:06:10] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 20:07:31,907 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:07:31,908 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:07:31,909 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:07:31] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:07:31,931 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:07:31,932 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:07:31] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:07:42,174 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:07:42,175 - WARNING - [app.py:228] - Error processing photo: encoding
2025-02-01 20:07:42,176 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:07:42] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:07:42,197 - ERROR - [app.py:187] - Error serving static file vcards/assets/images/default-pic.png: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-02-01 20:07:42,198 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:07:42] "[33mGET /vcards/assets/images/default-pic.png HTTP/1.1[0m" 404 -
2025-02-01 20:08:42,922 - INFO - [_internal.py:224] -  * Detected change in '/var/www/html/vCard/app.py', reloading
2025-02-01 20:08:42,955 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:08:43,097 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:08:43,097 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:08:43,097 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:08:43,097 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:08:43,098 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:08:43,098 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:08:43,098 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:08:43,102 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:08:43,103 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:09:27,390 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:09:27,390 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:09:27,390 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:09:27,390 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:09:27,391 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:09:27,391 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:09:27,391 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:09:27,396 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 20:09:27,397 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:09:27,526 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:09:27,526 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:09:27,526 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:09:27,526 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:09:27,527 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:09:27,527 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:09:27,527 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:09:27,531 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:09:27,532 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:09:29,528 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:29] "GET / HTTP/1.1" 200 -
2025-02-01 20:09:29,561 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:29] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:09:29,567 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:29] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:09:29,568 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:29] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:09:29,578 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:29] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:09:29,615 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:29] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:09:29,670 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:29] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:09:33,214 - INFO - [app.py:135] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:09:33,215 - INFO - [app.py:143] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:09:33,215 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:33] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:09:47,080 - INFO - [app.py:135] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:09:47,080 - INFO - [app.py:143] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:09:47,080 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:47] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:09:49,990 - INFO - [app.py:135] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:09:49,990 - INFO - [app.py:143] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:09:49,991 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:49] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:09:54,450 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:09:54,469 - WARNING - [app.py:234] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 20:09:54,470 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:09:54] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:10:18,166 - INFO - [app.py:171] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:10:18,171 - WARNING - [app.py:234] - Error processing photo: a bytes-like object is required, not 'str'
2025-02-01 20:10:18,173 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:10:18] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:14:18,849 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:14:18,849 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:14:18,849 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:14:18,849 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:14:18,850 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:14:18,850 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:14:18,850 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:14:18,856 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 20:14:18,856 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:14:18,974 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:14:18,974 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:14:18,974 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:14:18,974 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:14:18,975 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:14:18,975 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:14:18,975 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:14:18,979 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:14:18,979 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:14:25,768 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:25] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 20:14:25,793 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:25] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 20:14:25,803 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:25] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 20:14:25,804 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:25] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:14:25,822 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:25] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:14:25,839 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:25] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:14:25,921 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:25] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:14:28,969 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:14:28,969 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:14:28,970 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:28] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:14:48,643 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:14:48,643 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:14:48,643 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:48] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:14:51,551 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:14:51,551 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:14:51,551 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:14:51] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:15:38,116 - INFO - [app.py:194] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:15:38,136 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:15:38] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:19:38,812 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:19:38,813 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:19:38,813 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:19:38,813 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:19:38,814 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:19:38,814 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:19:38,814 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:19:38,819 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 20:19:38,819 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:19:38,937 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:19:38,938 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:19:38,938 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:19:38,938 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:19:38,939 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:19:38,939 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:19:38,939 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:19:38,943 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:19:38,943 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:19:44,240 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:44] "GET / HTTP/1.1" 200 -
2025-02-01 20:19:44,270 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:44] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:19:44,270 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:44] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:19:44,271 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:44] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:19:44,283 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:44] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:19:44,330 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:44] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:19:44,384 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:44] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:19:47,902 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:19:47,902 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:19:47,902 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:47] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:19:57,301 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:19:57,301 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:19:57,302 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:57] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:19:59,685 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:19:59,685 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:19:59,685 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:19:59] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:20:03,702 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:20:03,702 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:20:03,703 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:20:03] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:20:07,603 - INFO - [app.py:194] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:20:07,626 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:20:07] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:22:21,764 - INFO - [app.py:194] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:22:21,771 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:22:21] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:24:38,474 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:24:38,474 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:24:38,474 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:24:38,474 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:24:38,475 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:24:38,475 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:24:38,475 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:24:38,481 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 20:24:38,481 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:24:38,606 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:24:38,606 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:24:38,606 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:24:38,606 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:24:38,607 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:24:38,607 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:24:38,607 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:24:38,611 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:24:38,611 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:24:42,198 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:42] "GET / HTTP/1.1" 200 -
2025-02-01 20:24:42,229 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:42] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:24:42,238 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:42] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:24:42,239 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:42] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:24:42,257 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:42] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:24:42,285 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:42] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:24:42,332 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:42] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:24:50,013 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:24:50,013 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:24:50,014 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:24:50] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:25:00,510 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:25:00,510 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:25:00,510 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:25:00] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:25:01,860 - INFO - [app.py:194] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:25:01,882 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:25:01] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:26:13,073 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:26:13,073 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:26:13,073 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:26:13,073 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:26:13,074 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:26:13,074 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:26:13,074 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:26:13,079 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 20:26:13,079 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:26:13,198 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:26:13,198 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:26:13,198 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:26:13,198 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:26:13,199 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:26:13,199 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:26:13,199 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:26:13,203 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:26:13,203 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:26:32,723 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:32] "GET / HTTP/1.1" 200 -
2025-02-01 20:26:32,743 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:32] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:26:32,746 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:32] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:26:32,746 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:32] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:26:32,760 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:32] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:26:32,790 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:32] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:26:32,794 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:32] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:26:35,948 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:26:35,948 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:26:35,949 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:35] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:26:49,911 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:26:49,911 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:26:49,912 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:49] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:26:59,631 - INFO - [app.py:194] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:26:59,655 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:26:59] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:28:44,958 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:28:44] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 20:28:44,989 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:28:44] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 20:28:44,994 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:28:44] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 20:28:44,996 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:28:44] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:28:45,013 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:28:45] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:28:45,037 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:28:45] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:28:45,199 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:28:45] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:29:08,592 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:08,592 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:08,593 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:29:08] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:29:09,738 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:09,738 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:09,738 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:29:09] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:29:14,241 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:14,242 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:14,242 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:29:14] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:29:23,597 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:23,597 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:23,597 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:29:23] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:29:37,685 - INFO - [app.py:194] - Attempting to serve vCard webpage for: UmFtaSBoYW1pZCsy.vcf
2025-02-01 20:29:37,692 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:29:37] "GET /vcards/UmFtaSBoYW1pZCsy.vcf HTTP/1.1" 200 -
2025-02-01 20:41:18,773 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:41:18,773 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:41:18,773 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:41:18,773 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:41:18,774 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:41:18,774 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:41:18,774 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:41:18,780 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.98:8000 (Press CTRL+C to quit)
2025-02-01 20:41:18,780 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-01 20:41:18,899 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:41:18,899 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:41:18,899 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:41:18,899 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:41:18,900 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:41:18,900 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:41:18,900 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:41:18,904 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-01 20:41:18,905 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-01 20:41:30,563 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:30] "GET / HTTP/1.1" 200 -
2025-02-01 20:41:30,583 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:30] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:41:30,588 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:30] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:41:30,589 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:30] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:41:30,604 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:30] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:41:30,637 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:30] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:41:30,697 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:30] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:41:34,540 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:41:34,540 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:41:34,540 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:34] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:41:50,709 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:41:50,709 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:41:50,709 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:50] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:41:53,841 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:41:53,863 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:53] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:41:58,196 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:41:58,207 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:41:58] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:42:19,138 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:42:19,145 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:42:19] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:44:15,154 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:44:15,154 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:44:15,154 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:44:15,154 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:44:15,156 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:44:15,156 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:44:15,156 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:44:15,161 - INFO - [_internal.py:224] -  * Running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2025-02-01 20:44:27,535 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:27] "GET / HTTP/1.1" 200 -
2025-02-01 20:44:27,560 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:27] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:44:27,565 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:27] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:44:27,566 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:27] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:44:27,587 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:27] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:44:27,616 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:27] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:44:27,662 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:27] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:44:38,492 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:44:38,492 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:44:38,493 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:38] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:44:47,225 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:44:47,225 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QnJhaGltIEFpdG91.vcf
2025-02-01 20:44:47,226 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:47] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:44:50,209 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QnJhaGltIEFpdG91.vcf
2025-02-01 20:44:50,234 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:44:50] "GET /vcards/QnJhaGltIEFpdG91.vcf HTTP/1.1" 200 -
2025-02-01 20:48:39,430 - INFO - [app.py:30] - Current User: pc
2025-02-01 20:48:39,430 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-01 20:48:39,430 - INFO - [app.py:32] - Process User ID: 1000
2025-02-01 20:48:39,430 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-01 20:48:39,431 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-01 20:48:39,431 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-01 20:48:39,431 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-01 20:48:50,690 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:48:50] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 20:48:50,713 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:48:50] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 20:48:50,722 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:48:50] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-01 20:48:50,723 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:48:50] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:48:50,742 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:48:50] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:48:50,764 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:48:50] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:48:51,185 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:48:51] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:49:12,983 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:12] "GET / HTTP/1.1" 200 -
2025-02-01 20:49:13,003 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:13] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:49:13,008 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:13] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:49:13,012 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:13] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:49:13,018 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:13] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:49:13,047 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:13] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:49:13,100 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:13] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:49:21,844 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:21,844 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:21,845 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:21] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:49:24,709 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:24,709 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:24,709 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:24] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:49:25,984 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:25,984 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:25,985 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:25] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:49:29,482 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:29,482 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:29,482 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:29] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:49:33,598 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:33,599 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:33,599 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:33] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:49:46,669 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:46,669 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:46,670 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:46] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:49:51,160 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:51,161 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:49:51,161 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:49:51] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:50:09,128 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:50:09,135 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:50:09] "GET /vcards/QkVMQUZLSUggU2Fs.vcf HTTP/1.1" 200 -
2025-02-01 20:52:33,122 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:52:33,130 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:52:33] "GET /vcards/QkVMQUZLSUggU2Fs.vcf HTTP/1.1" 200 -
2025-02-01 20:52:41,348 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:52:41,355 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:52:41] "GET /vcards/QkVMQUZLSUggU2Fs.vcf HTTP/1.1" 200 -
2025-02-01 20:52:56,696 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QkVMQUZLSUggU2Fs.vcf
2025-02-01 20:52:56,702 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:52:56] "GET /vcards/QkVMQUZLSUggU2Fs.vcf HTTP/1.1" 200 -
2025-02-01 20:53:00,388 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:00] "GET / HTTP/1.1" 200 -
2025-02-01 20:53:00,419 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:00] "GET /assets/css/styles.css HTTP/1.1" 200 -
2025-02-01 20:53:00,422 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:00] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:53:00,427 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:00] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:53:00,435 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:00] "GET /assets/js/image-upload.js HTTP/1.1" 200 -
2025-02-01 20:53:00,471 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:00] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:53:00,518 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:00] "GET /assets/js/QRcode.js HTTP/1.1" 200 -
2025-02-01 20:53:13,782 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QUlUIE9VRktJUiBC.vcf
2025-02-01 20:53:13,783 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QUlUIE9VRktJUiBC.vcf
2025-02-01 20:53:13,783 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:13] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:53:22,896 - INFO - [app.py:172] - Attempting to save vCard to: /var/www/html/vCard/vcards/QUlUIE9VRktJUiBC.vcf
2025-02-01 20:53:22,896 - INFO - [app.py:183] - Successfully saved vCard: /var/www/html/vCard/vcards/QUlUIE9VRktJUiBC.vcf
2025-02-01 20:53:22,897 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:22] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-01 20:53:25,528 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QUlUIE9VRktJUiBC.vcf
2025-02-01 20:53:25,564 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:53:25] "GET /vcards/QUlUIE9VRktJUiBC.vcf HTTP/1.1" 200 -
2025-02-01 20:54:01,562 - INFO - [app.py:211] - Attempting to serve vCard webpage for: QUlUIE9VRktJUiBC.vcf
2025-02-01 20:54:01,602 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:01] "GET /vcards/QUlUIE9VRktJUiBC.vcf HTTP/1.1" 200 -
2025-02-01 20:54:05,050 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:05] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-01 20:54:05,078 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:05] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-01 20:54:05,081 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:05] "GET /assets/images/default-pic.png HTTP/1.1" 200 -
2025-02-01 20:54:05,083 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:05] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:54:05,089 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:05] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-01 20:54:05,122 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:05] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-01 20:54:05,129 - INFO - [_internal.py:224] - 127.0.0.1 - - [01/Feb/2025 20:54:05] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-03 09:48:44,052 - INFO - [app.py:30] - Current User: pc
2025-02-03 09:48:44,052 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 09:48:44,052 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 09:48:44,052 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 09:48:44,053 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 09:48:44,053 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 09:48:44,053 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 09:48:44,059 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.70:8000 (Press CTRL+C to quit)
2025-02-03 09:48:44,059 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-03 09:48:44,175 - INFO - [app.py:30] - Current User: pc
2025-02-03 09:48:44,175 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 09:48:44,175 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 09:48:44,175 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 09:48:44,176 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 09:48:44,176 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 09:48:44,176 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 09:48:44,181 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-03 09:48:44,181 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-03 09:48:51,881 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:51] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-03 09:48:51,904 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:51] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-03 09:48:51,907 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:51] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-03 09:48:51,908 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:51] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-03 09:48:51,917 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:51] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-03 09:48:51,944 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:51] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-03 09:48:52,010 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:52] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-03 09:48:55,249 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:48:55] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-03 09:49:20,741 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/SGFtemEgcmFtaSsy.vcf
2025-02-03 09:49:20,741 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/SGFtemEgcmFtaSsy.vcf
2025-02-03 09:49:20,741 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:49:20] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 09:49:25,013 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/SGFtemEgcmFtaSsy.vcf
2025-02-03 09:49:25,013 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/SGFtemEgcmFtaSsy.vcf
2025-02-03 09:49:25,014 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:49:25] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 09:49:47,328 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/SGFtemEgcmFtaSsy.vcf
2025-02-03 09:49:47,328 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/SGFtemEgcmFtaSsy.vcf
2025-02-03 09:49:47,328 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:49:47] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 09:50:19,239 - INFO - [app.py:194] - Attempting to serve vCard webpage for: SGFtemEgcmFtaSsy.vcf
2025-02-03 09:50:19,264 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:50:19] "GET /vcards/SGFtemEgcmFtaSsy.vcf HTTP/1.1" 200 -
2025-02-03 09:50:22,616 - INFO - [app.py:194] - Attempting to serve vCard webpage for: SGFtemEgcmFtaSsy.vcf
2025-02-03 09:50:22,626 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:50:22] "GET /vcards/SGFtemEgcmFtaSsy.vcf HTTP/1.1" 200 -
2025-02-03 09:53:45,876 - INFO - [app.py:194] - Attempting to serve vCard webpage for: SGFtemEgcmFtaSsy.vcf
2025-02-03 09:53:45,881 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:53:45] "GET /vcards/SGFtemEgcmFtaSsy.vcf HTTP/1.1" 200 -
2025-02-03 09:53:51,001 - INFO - [app.py:194] - Attempting to serve vCard webpage for: QUlUIE9VRktJUiBC.vcf
2025-02-03 09:53:51,045 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 09:53:51] "GET /vcards/QUlUIE9VRktJUiBC.vcf HTTP/1.1" 200 -
2025-02-03 10:01:50,479 - INFO - [_internal.py:224] -  * Detected change in '/var/www/html/vCard/app.py', reloading
2025-02-03 10:01:50,512 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-03 10:01:50,642 - INFO - [app.py:30] - Current User: pc
2025-02-03 10:01:50,642 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 10:01:50,642 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 10:01:50,642 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 10:01:50,643 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 10:01:50,644 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 10:01:50,644 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 10:01:50,648 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-03 10:01:50,648 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-03 10:02:45,816 - INFO - [app.py:30] - Current User: pc
2025-02-03 10:02:45,816 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 10:02:45,816 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 10:02:45,816 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 10:02:45,817 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 10:02:45,817 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 10:02:45,817 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 10:02:45,823 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.70:8000 (Press CTRL+C to quit)
2025-02-03 10:02:45,823 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-03 10:02:45,942 - INFO - [app.py:30] - Current User: pc
2025-02-03 10:02:45,942 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 10:02:45,942 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 10:02:45,942 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 10:02:45,943 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 10:02:45,943 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 10:02:45,943 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 10:02:45,947 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-03 10:02:45,947 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-03 10:02:54,214 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:02:54] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-03 10:02:54,248 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:02:54] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-03 10:02:54,250 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:02:54] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-03 10:02:54,253 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:02:54] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-03 10:02:54,261 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:02:54] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-03 10:02:54,288 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:02:54] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-03 10:02:54,370 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:02:54] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-03 10:03:20,245 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:20,246 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:20,246 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:20] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:23,567 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:23,567 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:23,568 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:23] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:26,274 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:26,274 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:26,274 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:26] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:27,454 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:27,455 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:27,455 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:27] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:28,947 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:28,947 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:28,947 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:28] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:36,011 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:36,012 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:36,012 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:36] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:48,645 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:48,645 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:48,645 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:48] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:52,124 - INFO - [app.py:158] - Attempting to save vCard to: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:52,124 - INFO - [app.py:166] - Successfully saved vCard: /var/www/html/vCard/vcards/TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:52,125 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:52] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:03:55,150 - INFO - [app.py:194] - Attempting to serve vCard webpage for: TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:55,172 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:55] "GET /vcards/TGFoeWFubmkgWW91.vcf HTTP/1.1" 200 -
2025-02-03 10:03:57,012 - INFO - [app.py:194] - Attempting to serve vCard webpage for: TGFoeWFubmkgWW91.vcf
2025-02-03 10:03:57,021 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:03:57] "GET /vcards/TGFoeWFubmkgWW91.vcf HTTP/1.1" 200 -
2025-02-03 10:06:03,892 - INFO - [app.py:30] - Current User: pc
2025-02-03 10:06:03,892 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 10:06:03,892 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 10:06:03,892 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 10:06:03,893 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 10:06:03,893 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 10:06:03,893 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 10:06:03,898 - INFO - [_internal.py:224] -  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:8000
 * Running on http://192.168.1.70:8000 (Press CTRL+C to quit)
2025-02-03 10:06:03,899 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-03 10:06:04,040 - INFO - [app.py:30] - Current User: pc
2025-02-03 10:06:04,040 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 10:06:04,040 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 10:06:04,040 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 10:06:04,041 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 10:06:04,041 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 10:06:04,041 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 10:06:04,046 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-03 10:06:04,046 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
2025-02-03 10:06:07,082 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:07] "[36mGET / HTTP/1.1[0m" 304 -
2025-02-03 10:06:07,110 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:07] "[36mGET /assets/css/styles.css HTTP/1.1[0m" 304 -
2025-02-03 10:06:07,119 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:07] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-03 10:06:07,120 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:07] "[36mGET /assets/images/default-pic.png HTTP/1.1[0m" 304 -
2025-02-03 10:06:07,142 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:07] "[36mGET /assets/js/image-upload.js HTTP/1.1[0m" 304 -
2025-02-03 10:06:07,157 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:07] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-03 10:06:08,419 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:08] "[36mGET /assets/js/QRcode.js HTTP/1.1[0m" 304 -
2025-02-03 10:06:12,240 - INFO - [app.py:93] - Profile file received: None
2025-02-03 10:06:12,240 - WARNING - [app.py:120] - No profile file or filename found. Using default image.
2025-02-03 10:06:12,240 - INFO - [app.py:169] - Attempting to save vCard to: /var/www/html/vCard/vcards/QUlUIE9VRktJUiBC.vcf
2025-02-03 10:06:12,240 - INFO - [app.py:177] - Successfully saved vCard: /var/www/html/vCard/vcards/QUlUIE9VRktJUiBC.vcf
2025-02-03 10:06:12,241 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:12] "POST /generate_vcard HTTP/1.1" 200 -
2025-02-03 10:06:17,538 - INFO - [app.py:205] - Attempting to serve vCard webpage for: QUlUIE9VRktJUiBC.vcf
2025-02-03 10:06:17,563 - INFO - [_internal.py:224] - 127.0.0.1 - - [03/Feb/2025 10:06:17] "GET /vcards/QUlUIE9VRktJUiBC.vcf HTTP/1.1" 200 -
2025-02-03 10:10:21,759 - INFO - [_internal.py:224] -  * Detected change in '/var/www/html/vCard/app.py', reloading
2025-02-03 10:10:21,785 - INFO - [_internal.py:224] -  * Restarting with stat
2025-02-03 10:10:21,904 - INFO - [app.py:30] - Current User: pc
2025-02-03 10:10:21,904 - INFO - [app.py:31] - Current Working Directory: /var/www/html/vCard
2025-02-03 10:10:21,904 - INFO - [app.py:32] - Process User ID: 1000
2025-02-03 10:10:21,904 - INFO - [app.py:33] - Process Group ID: 1000
2025-02-03 10:10:21,905 - INFO - [app.py:47] - Attempting to create vCard storage directory: /var/www/html/vCard/vcards
2025-02-03 10:10:21,905 - ERROR - [app.py:57] - Could not change vcards directory permissions: [Errno 1] Operation not permitted: '/var/www/html/vCard/vcards'
2025-02-03 10:10:21,905 - ERROR - [app.py:58] - Current directory permissions: 0o777
2025-02-03 10:10:21,909 - WARNING - [_internal.py:224] -  * Debugger is active!
2025-02-03 10:10:21,909 - INFO - [_internal.py:224] -  * Debugger PIN: 119-535-239
